featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -551,15 +552,21 @@ def _construct_count_stats_sql(
         expressions.Select
         """
         cat_counts = self._get_cat_counts(col_expr)
+        col_expr_filled_null = expressions.Case(
+            ifs=[
+                expressions.If(
+                    this=expressions.Is(this=col_expr, expression=expressions.Null()),
+                    true=make_literal_value(MISSING_VALUE_REPLACEMENT),
+                )
+            ],
+            default=col_expr,
+        )
```

Comment:
Is this existing edge case (when the value is null) that is not covered by current implementation?

Reply:
Yes, without this missing values are ignored in the counts.